src/Kestrel.Core/Internal/Infrastructure/CorrelationIdGenerator.cs

```diff
@@ -3,13 +3,19 @@
 
 using System;
 using System.Threading;
+using System.Runtime.CompilerServices;
+
+#if NETCOREAPP2_1
+using System.Buffers;
+using System.Runtime.InteropServices;
+#endif
```

Comment:
Does this need to be ifdefed?

Reply:
No.

It's an artefact from my personal code-style. Shall I remove this?

Reply:
Not a big deal, but yes.